book/src/lint_configuration.md

```diff
@@ -472,6 +472,16 @@ Whether to apply the raw pointer heuristic to determine if a type is `Send`.
 * [`non_send_fields_in_send_ty`](https://rust-lang.github.io/rust-clippy/master/index.html#non_send_fields_in_send_ty)
 
 
+## `enable-source-item-ordering-for`
```

Comment:
Might be better `source-item-ordering` for consistence with `module-item-order-groupings`

Reply:
Sounds good to me. Naming changed and fixup pushed for further review.